governance/polls/Ratification Poll - Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Ratification Poll for Governance Communications Core Unit, COM-001, Mandate Refresh (MIP39c2-SP30) - April 11, 2022
+summary: This poll is for accepting the updates to the mandate for Core Unit COM-001.
```

Comment:
"This subproposal refreshes the mandate for the Governance Communications Core Unit (COM-001)."